src/vstart.sh

```diff
@@ -733,6 +733,9 @@ $DAEMONOPTS
         mds root ino gid = `id -g`
         $(format_conf "${extra_conf}")
 [mgr]
+        mgr initial modules = $mgr_modules
```

Comment:
`mgr_initial_modules` is consumed by mon,

```console
$ git grep mgr_initial_modules
...
src/mon/MgrMonitor.cc:  auto initial_modules = g_conf().get_val<std::string>("mgr_initial_modules");
...
```

Reply:
> 
> 
> `mgr_initial_modules` is consumed by mon,
> 
> ```
> $ git grep mgr_initial_modules
> ...
> src/mon/MgrMonitor.cc:  auto initial_modules = g_conf().get_val<std::string>("mgr_initial_modules");
> ...
> ```

Right, 
My PR was based on the effect of modifications made to `ceph.conf`.
So I'll recheck. 

But still rather odd, and unexpected, that such a config would be in the `[mon]` config.